doc/decoding/going_further.rst

```diff
@@ -0,0 +1,276 @@
+    cross-validation schemes, see:
+    <https://scikit-learn.org/stable/modules/cross_validation.html>`_
+  * `how to evaluate a model using scikit-learn
+    <http://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values>`_
+
+
+Measuring the chance level
+----------------------------------
+
+**Dummy estimators**: The simplest way to measure prediction performance
+at chance, is to use a *"dummy"* classifier,
+:class:`sklearn.dummy.DummyClassifier` (purely random)::
+
+    >>> from sklearn.dummy import DummyClassifier
+    >>> null_cv_scores = cross_val_score(DummyClassifier(), fmri_masked, conditions, cv=cv)  # doctest: +SKIP
+
+**Permutation testing**: A more controlled way, but slower, is to do
+permutation testing on the labels, with
+:func:`sklearn.model_selection.permutation_test_score`::
+
+    >>> from sklearn.model_selection import permutation_test_score
+    >>> null_cv_scores = permutation_test_score(svc, fmri_masked, conditions, cv=cv)  # doctest: +SKIP
+
+.. topic:: **Decoding on simulated data**
+
+   Simple simulations may be useful to understand the behavior of a given
+   decoder on data. In particular, simulations enable us to set the true
+   weight maps and compare them to the ones retrieved by decoders. A full
+   example running simulations and discussing them can be found in
+   :ref:`sphx_glr_auto_examples_02_decoding_plot_simulated_data.py`
+   Simulated data cannot easily mimic all properties of brain data. An
+   important aspect, however, is its spatial structure, that we create in
+   the simulations.
+
+
+Decoding without a mask: Anova-SVM in scikit-lean
+==================================================
+
+We can also implement feature selection before decoding as a scikit-learn
+`pipeline`(:class:`sklearn.pipeline.Pipeline`). For this, we need to import
+the :mod:`sklearn.feature_selection` module and use
+:func:`sklearn.feature_selection.f_classif`, a simple F-score
+based feature selection (a.k.a. `Anova <https://en.wikipedia.org/wiki/Analysis_of_variance#The_F-test>`_),
+
+    >>> from sklearn.feature_selection import SelectPercentile, f_classif
+    >>> feature_selection = SelectPercentile(f_classif, percentile=5)
+    >>> from sklearn.pipeline import Pipeline
+    >>> anova_svc = Pipeline([('anova', feature_selection), ('svc', svc)])
+    >>> # We can use our ``anova_svc`` object exactly as we were using our ``svc``
+    >>> # object previously.
+    >>> cv_scores = cross_val_score(anova_svc, fmri_masked, conditions,
+                                    cv=cv, groups=session_label) # doctest: +SKIP
+    >>> print(cv_scores.mean()) # doctest: +SKIP
+    >>> # Visualize the SVC's discriminating weights
+    >>> coef = svc.coef_ # doctest: +SKIP
+    >>> coef = feature_selection.inverse_transform(coef) # doctest: +SKIP
+    >>> weight_img = masker.inverse_transform(coef) # doctest: +SKIP
+    >>> plot_stat_map(weight_img, title='Anova+SVC weights') # doctest: +SKIP
```

Comment:
This piece of code should rather be an example

Reply:
I removed all these and created a dedicated example in the 07_advanced section. It's also in the PR as it is tightly related